src/_modules/experimental/Request.sol

```diff
@@ -173,7 +173,9 @@ library LibRequestBuilder {
             return RequestError.commandFailed();
```

Comment:
Should we bubble up the error instead?

Reply:
see https://github.com/nomoixyz/vulcan/pull/179/files#diff-10e9fe0036fe51bff69b302ccb16092d5fac57335019f2d471be3c7d07845d7dR163